helm-charts/helm-api/ibm_cloud_pak/manifest.yaml

```diff
@@ -1,111 +1,142 @@
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    pull-repository: quay.io/opencloudio/icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    platform:
+      os: linux
+      arch: s390x
 
-- image: icp-helm-rudder:3.2.2
+- image: tiller:v2.16.1-cs-3.3.0
   references:	
-  - repository : icp-helm-rudder-amd64:3.2.2
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-amd64/icp-helm-rudder:3.2.2
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: amd64
-  - repository : icp-helm-rudder-ppc64le:3.2.2
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-ppc64le/icp-helm-rudder:3.2.2
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: ppc64le
-  - repository : icp-helm-rudder-s390x:3.2.2
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-s390x/icp-helm-rudder:3.2.2
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: s390x
\ No newline at end of file
+  - repository : tiller:v2.16.1-cs-3.3.0-amd64
+    pull-repository: quay.io/opencloudio/tiller:v2.16.1-cs-3.3.0-amd64
+    platform:
+      os: linux
+      arch: amd64
+  - repository : tiller:v2.16.1-cs-3.3.0-ppc64le
+    pull-repository: quay.io/opencloudio/tiller:v2.16.1-cs-3.3.0-ppc64le
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : tiller:v2.16.1-cs-3.3.0-s390x
+    pull-repository: quay.io/opencloudio/tiller:v2.16.1-cs-3.3.0-s390x
+    platform:
+      os: linux
+      arch: s390x
+- image: tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+  references:	
+  - repository : tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    pull-repository: quay.io/opencloudio/tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    platform:
+      os: linux
+      arch: amd64
+  - repository : tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    pull-repository: quay.io/opencloudio/tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    pull-repository: quay.io/opencloudio/tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
```

Comment:
tiller is at v2.16.1-cs-3.3.1 and digest is `a7539700cdc49623efd9c049760d9b5c32329c498be55f30a87835afdc43fe47`